modules/understanding-update-channels.adoc

```diff
@@ -75,7 +75,7 @@ Red Hat monitors newly released versions and update paths associated with those
 
 If Red Hat removes update recommendations from any supported release, a superseding update recommendation will be provided to a future version that corrects the regression. There may however be a delay while the defect is corrected, tested, and promoted to your selected channel.
 
-Beginning in {product-title} 4.10, when update risks are confirmed, they are declared as Conditional Update risks for the relevant updates. Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those updates are supported but not recommended, and a reference link is provided. The reference link helps the cluster admin decide if they would like to accept the risk and update anyway.
+Beginning in {product-title} 4.10, when update risks are confirmed, they are declared as Conditional Update risks for the relevant updates. Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those update paths are labeled as updates with *known issues*, and a reference link to the known issues is provided. The reference link helps the cluster admin decide if they want to accept the risk and continue to upgrade their cluster.
```

Comment:
Just something for your consideration - in this [ISG table](https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element), "words and phrases that are defined or explained" should have italics rather than bold:
```suggestion
Beginning in {product-title} 4.10, when update risks are confirmed, they are declared as Conditional Update risks for the relevant updates. Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those update paths are labeled as updates with _known issues_, and a reference link to the known issues is provided. The reference link helps the cluster admin decide if they want to accept the risk and continue to upgrade their cluster.
```

I'm less sure of this part but also maybe the phrase we're attempting to define here is "updates with known issues" and not just the known issues part. Up to you, but it's just a thought:
```suggestion
Beginning in {product-title} 4.10, when update risks are confirmed, they are declared as Conditional Update risks for the relevant updates. Each known risk may apply to all clusters or only clusters matching certain conditions. Some examples include having the `Platform` set to `None` or the CNI provider set to `OpenShiftSDN`. The Cluster Version Operator (CVO) continually evaluates known risks against the current cluster state. If no risks match, the update is recommended. If the risk matches, those update paths are labeled as _updates with known issues_, and a reference link to the known issues is provided. The reference link helps the cluster admin decide if they want to accept the risk and continue to upgrade their cluster.
```

Reply:
I think I like the latter - ty!